pennylane/tape/tape.py

```diff
@@ -80,6 +81,24 @@
 """
 
 
+def get_tape():
```

Comment:
Maybe `get_active_tape` instead?

Reply:
Good call, I almost named it this but changed last minute!